### PR TITLE
fix(RHICOMPL-880): Update Ansible icon to use patternfly version

### PR DESCRIPTION
**Pull request by bastilian**

### CODE REVIEW

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 1]
```diff
@@ -14,6 +14,15 @@ import {
 import { HIGH_SEVERITY, MEDIUM_SEVERITY, LOW_SEVERITY } from './Constants';
 import ComplianceRemediationButton from './ComplianceRemediationButton';
 
+export const columns = [
+    { title: 'Rule', transforms: [ sortable ] },
+    { title: 'Policy', transforms: [ sortable ] },
+    { title: 'Severity', transforms: [ sortable ] },
+    { title: 'Passed', transforms: [ sortable ] },
+    { title: <React.Fragment><Ansible size='lg'/> Ansible</React.Fragment>,
```
Comment (vkrizan) [inline packages/inventory-compliance/src/SystemRulesTable.js:22]: The icon is too big compared to the text. Try medium, or maybe the default would inherit font-size.

Screenshot_2020-10-02 Compliance

File: packages/inventory-compliance/src/SystemRulesTable.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import { Pagination, PaginationVariant, ToolbarItem } from '@patternfly/react-core';
-import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
-import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon, AnsibeTowerIcon as Ansible } from '@patternfly/react-icons';
```
Comment (vkrizan) [inline packages/inventory-compliance/src/SystemRulesTable.js:4]: What's the rationale on renaming the icon name?

**Review by vkrizan — CHANGES_REQUESTED** [review]

File: packages/inventory-compliance/src/SystemRulesTable.js (hunk 2 above)
Comment (bastilian) [reply packages/inventory-compliance/src/SystemRulesTable.js]: ... evening musings. :D I removed the alias.

File: packages/inventory-compliance/src/SystemRulesTable.js (hunk 1 above)
Comment (bastilian) [reply packages/inventory-compliance/src/SystemRulesTable.js]: Changed to 'sm' that should fit better.

Comment (vkrizan) [reply packages/inventory-compliance/src/SystemRulesTable.js]: I guess the medium would fit better, but I'll leave it to your discretion.

**Review by vkrizan — APPROVED** [review]

**Pull request merged**